DotNet UWP Samples/Earthquake Simulator/Earthquake Simulator/ViewModel/EarthQuakeViewModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
```

Comment:
Please add a detailed README.md 
Uber README is here - https://github.com/microsoft/DataStreamerSamples/blob/master/README.md